OSDOCS#10223: Consolidate "Installing from OperatorHub" procs & Single vs AllNamespace improvements (#79401)
openshift/openshift-docs: modules/olm-installing-from-operatorhub-using-web-console.adoc

.. For clusters on cloud providers with token authentication enabled:
+
--
* If the cluster uses {aws-short} {sts-short} (*STS Mode* in the web console), enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
+

> image::oadp-install-operator-role-arn.png[Entering the ARN]

Author: This screenshot was introduced as a result of a review request in https://github.com/openshift/openshift-docs/pull/66903#pullrequestreview-1731973852, however I'm removing it now for a few reasons:

* Primarily, we have style guidelines to avoid screenshots in general (https://redhat-documentation.github.io/supplementary-style-guide/#screenshots) and having the screenshot isn't required here.
* The matching Entra ID-related bullet does not have a matching screenshot, so it feels/looks out of place.
* This is the only image/screenshot in the entire assembly, so it feels even more out of place.

.. If you selected an *Automatic* approval strategy, the upgrade status should resolve to *Up to date* without intervention.

.Verification

> * After the upgrade status of the subscription is *Up to date*, select *Operators → Installed Operators* to verify that the cluster service version (CSV) of the installed Operator eventually shows up. The *Status* should ultimately resolve to *InstallSucceeded* in the relevant namespace.

Reviewer: Seems like it only shows the `Succeeded`, not the `InstallSucceeded`.
<img width="1684" alt="image" src="https://github.com/user-attachments/assets/4104c5de-7b35-4321-a5c7-8886f0e5c816">

+
If it does not:

> ** Check the logs in any pods in the `openshift-operators` project (or other relevant namespace if *A specific namespace...* installation mode was selected) on the *Workloads → Pods* page that are reporting issues to troubleshoot further.

Reviewer 2: ```suggestion
** Check the logs in any pods in the `openshift-operators` project (or other relevant namespace if you selected the `SingleNamespace` install  mode) on the *Workloads → Pods* page that are reporting issues to troubleshoot further.
```

Author: Leaving as-is because the web console doesn't refer to it as `SingleNamespace`.
